versioned_docs/version-v1.1/vm/backup-restore.md

```diff
@@ -140,7 +142,9 @@ VM snapshots are created from the **Virtual Machines** page. The VM snapshot vol
 
 **Result:** The snapshot is created. You can also go to the `Backup & Snapshot > VM Snapshots` page to view all VM snapshots.
 
-The `ReadyToUse` status will be set to `true` once the snapshot is complete.
+The `State` will be set to `Ready` once the snapshot is complete.
```

Comment:
@WuJun2016 can you please help to double-check if this is true for v1.1, thanks.

Reply:
They also apply to v1.1.